modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -132,6 +109,29 @@ def submit_power_of_attorney(poa_code, form_number)
           )
         end
 
+        def set_auth_headers
+          headers = auth_headers.merge!({ VA_NOTIFY_KEY => icn_for_vanotify })
+
+          if feature_enabled_and_claimant_present?
+            add_dependent_to_auth_headers(headers)
+          else
+            auth_headers
+          end
+        end
+
+        def add_dependent_to_auth_headers(headers)
+          claimant = user_profile.profile
+
+          headers.merge!({
+                           dependent: {
+                             participant_id: claimant.participant_ids[0],
+                             ssn: claimant.ssn,
+                             first_name: claimant.given_names[0],
```

Comment:
`given_names` is an array and I was not sure if there was any idea at all as far as what to select so just prioritized the first name, if anyone has any information on a better approach open to it. (same with the participant IDs above)

Reply:
Same as above (why not just pass through the service object instead?)